site2/docs/concepts-messaging.md

```diff
@@ -316,10 +316,11 @@ A subscription is a named configuration rule that determines how messages are de
 > * If you want to achieve "message queuing" among consumers, share the same subscription name among multiple consumers(shared, failover, key_shared).
 > * If you want to achieve both effects simultaneously, combine exclusive subscription type with other subscription types for consumers.
 
-### Consumerless Subscriptions and Their Corresponding Types
+### Subscriptions type
```

Comment:
Shall we use `Subscription types` instead?

Reply:
Ah, yes, thanks for reminding me, can you update it in your PR https://github.com/apache/pulsar/pull/12794? thanks